lib/avromatic/model/attributes.rb

```diff
@@ -81,6 +81,12 @@ def required?(field)
         end
 
         def avro_field_class(field_type)
+          if Avromatic.custom_types.key?(field_type.try(:fullname))
+            custom_type = Avromatic.custom_types[field_type.fullname]
+            value_class = custom_type.value_class
+            return value_class if value_class
```

Comment:
The way this is currently implemented, a value class is not required. You can specify a "custom type" based on an Avro fullname, then associate `to_avro`/`from_avro` Procs with that. The class associated with the Virtus attribute will then be based on the builtin Avro type.